validator/java/build_and_test.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+#
+# Copyright 2020 The AMP HTML Authors. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS-IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the license.
+#
+# This script builds the Java AMP Validator and runs a full test suite.
+
+echo travis_fold:start:java_validator_build && \
+  ./build.sh && \
+  echo travis_fold:end:java_validator_build && \
+  bazel run //:amphtml_validator_test
```

Comment:
Same here (commands can be run separately)

Reply:
Done